icevision/models/torchvision_models/faster_rcnn/prediction.py

```diff
@@ -61,9 +65,25 @@ def convert_raw_prediction(raw_pred: dict, detection_threshold: float):
         bbox = BBox.from_xyxy(*xyxy)
         bboxes.append(bbox)
 
-    return {
+    d = {
         "labels": labels,
         "scores": scores,
         "bboxes": bboxes,
         "above_threshold": above_threshold,
     }
+
+    if raw_pred.get("keypoints") is not None:
```

Comment:
Here we should also implement the specific functionality into `keypoint_rcnn/prediction.py`, in that way we also don't need the `if raw_pred.get("keypoints") is not None:`

Reply:
Done! 
Also added a test on predictions, which I did not realize was missing.